google/auth/transport/requests.py

```diff
@@ -150,27 +150,31 @@ class AuthorizedSession(requests.Session):
             refresh the credentials and retry the request.
         refresh_timeout (Optional[int]): The timeout value in seconds for
             credential refresh HTTP requests.
-        kwargs: Additional arguments passed to the :class:`requests.Session`
-            constructor.
+        auth_request_session (:class:`requests.Session`): (Optional)
+            HTTP object to make requests.
```

Comment:
Instead of `HTTP object to make requests.` can you update this to `A Requests session used when refreshing credentials.`?

Also would it make sense for this to just accept a `google.auth.transport.Request` object instead?

Reply:
:+1:

Hmm, from a user perspective it would be easier for me to just pass in the `Session` object. If it accepts a `Request` object, it'd just add an additional step of wrapping the `Session` in a `Request` object. I think both options are sensible though, so I'll defer to you to decide.

Reply:
Looking over the other implementations I think it makes the most sense to make this a Request object.

Reply:
Sounds good!